openedx/features/learner_profile/__init__.py

```diff
@@ -8,12 +8,16 @@
 # Namespace for learner profile waffle flags.
 WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='learner_profile')
 
-# Waffle flag for showing a message about the new profile features.
-# TODO: LEARNER-2554: 09/2017: Remove flag once message is no longer needed.
-SHOW_PROFILE_MESSAGE = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_message')
-
-# Waffle flag to show achievements on the learner profile.
-SHOW_ACHIEVEMENTS_FLAG = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_achievements', flag_undefined_default=True)
-
 # Waffle flag to redirect to another learner profile experience.
+# .. toggle_name: REDIRECT_TO_PROFILE_MICROFRONTEND
+# .. toggle_type: waffle_flag
+# .. toggle_default: False
+# .. toggle_description: Supports staged rollout of a new micro-frontend-based implementation of the profile page.
+# .. toggle_category: micro-frontend
+# .. toggle_use_cases: incremental_release, open_edx
+# .. toggle_creation_date: 2019-02-19
+# .. toggle_expiration_date: 2020-12-31
```

Comment:
Note: the date seems so arbitrary, I wonder what purpose it serves. Would it be better to have some milestone (maybe a named release), or is the DEPR ticket good enough?

Reply:
I can add a comment on the DEPR ticket to update this annotation once we are closer to and have a better understanding of the date.